### PR TITLE
add information on GTP item dropping to /info

### PR DESCRIPTION
**Pull request by beholderface**
also includes a bit about maximum range since that often comes up when someone is asking why the spell isn't working and their items are spilling all over the place

### CODE REVIEW

File: src/HexBug/cogs/info.py [hunk 1]
```diff
@@ -107,6 +107,12 @@ class InfoMessages(Enum):
         content="https://media.hexxy.media/data/great_spells.zip"
     )
 
+    gtp_itemdrop = MessageProps(
```
Comment (object-Object) [inline src/HexBug/cogs/info.py:110]: This should use an embed, like `bug_report` or `forum`.

File: src/HexBug/cogs/info.py [hunk 2]
```diff
@@ -107,6 +107,12 @@ class InfoMessages(Enum):
...
+        content="""When someone casts Greater Teleport on themself, each item in their inventory (except for those in their hands and any slots added by mods) has a random chance to drop at their destination.
+        For items in the main three inventory rows, the chance is N/10,000. For items in their hotbar, the chance is N/20,000. For items in their armor slots, the chance is N/40,000.
```
Comment (object-Object) [inline src/HexBug/cogs/info.py:112]: This needs to be dedented (see `bug_report`) - otherwise the indentation will show up in the actual string.

**Review by object-Object — CHANGES_REQUESTED** [review]

**Review by beholderface — COMMENTED** [review]
ok

**Review by object-Object — APPROVED** [review]
thank

**Pull request merged**